### PR TITLE
Add UBC Grades Integration

### PR DESCRIPTION
**Pull request by StashC**
Addresses #33 
Added grades information to the popup.
Will call the UBC grades api and display average infromation.
Also will form a link to UBC grades.

Make sure to test both UBC Vancouver and UBC O campus courses, as they have are under a different URI.

Some courses throw 404, Some I have confirmed are not on the UBC grades website.

### CODE REVIEW

File: src/content/SectionPopup/GradesComponent/GradesComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          <div className="GradesContainer">
+            <table>
+              <tr>
+                <td>Average (All Time)</td>
+                <td>{gradesData?.average.toFixed(2)}</td>
+              </tr>
+              <tr>
+                <td>Average (5 Years)</td>
+                <td>{gradesData?.averageFiveYears.toFixed(2)}</td>
+              </tr>
```
Comment (Rileyc2022) [inline src/content/SectionPopup/GradesComponent/GradesComponent.tsx:51]: Clicking sections without a five year average like AFST_V 350-A_101 breaks the extension, just shows a white page. Maybe show whatever's available:
```suggestion
              {gradesData?.average && <tr>
                <td>Average (All Time)</td>
                <td>{gradesData?.average.toFixed(2)}</td>
              </tr>}
              {gradesData?.averageFiveYears && <tr>
                <td>Average (5 Years)</td>
                <td>{gradesData?.averageFiveYears.toFixed(2)}</td>
              </tr>}
```

Comment (StashC) [reply src/content/SectionPopup/GradesComponent/GradesComponent.tsx:51]: will add null checks good catch

Comment (StashC) [reply src/content/SectionPopup/GradesComponent/GradesComponent.tsx]: @Rileyc2022 Pushed, can you try? 
image

Comment (Rileyc2022) [reply src/content/SectionPopup/GradesComponent/GradesComponent.tsx]: yeah tested lgtm

**Comment by Rileyc2022** [comment]
lgtm

**Pull request merged**